app/models/miq_alert.rb

```diff
@@ -193,16 +193,22 @@ def evaluate(target, inputs = {})
     # If we are alerting, invoke the alert actions, then add a status so we can limit how often to alert
     # Otherwise, destroy this alert's statuses for our target
     invoke_actions(target, inputs) if result
-    add_status_post_evaluate(target, result, inputs[:description])
-
+    add_status_post_evaluate(target, result, inputs[:ems_event])
     result
   end
 
-  def add_status_post_evaluate(target, result, status_description)
+  def add_status_post_evaluate(target, result, event)
+    status_description, severity, url = [
+      event.try(:message),
+      event.try(:full_data).try(:[], :severity),
+      event.try(:full_data).try(:[], :url),
+    ]
     status = miq_alert_statuses.find_or_initialize_by(:resource => target)
     status.result = result
     status.ems_id = target.try(:ems_id)
     status.description = status_description || description
```

Comment:
Actually this might interfere with existing alerts (change their message undesirably)
I can 
- only set it for events of a specific kind (`datawarehouse_event`)
- use some easly distinguishable metadata that won't be there by mistake, e.g
 ```event.try(:full_data).try(:[miq_alert_message],```